include/FFmpegReader.h

```diff
@@ -233,14 +233,13 @@ namespace openshot {
 		/// codecs have trouble seeking, and can introduce artifacts or blank images into the video.
 		bool enable_seek;
 
-		/// Constructor for FFmpegReader.  This automatically opens the media file and loads
-		/// frame 1, or it throws one of the following exceptions.
-		FFmpegReader(std::string path);
-
-		/// Constructor for FFmpegReader.  This only opens the media file to inspect its properties
-		/// if inspect_reader=true. When not inspecting the media file, it's much faster, and useful
-		/// when you are inflating the object using JSON after instantiating it.
-		FFmpegReader(std::string path, bool inspect_reader);
+		/// @brief Constructor for FFmpegReader.
+		///
+		/// Sets (and possibly opens) the media file path,
+		/// or throws an exception.
+		/// @param path  The filesystem location to load
+		/// @param inspect_reader  if true (the default), automatically open the media file and loads frame 1.
+		FFmpegReader(const std::string& path, bool inspect_reader=true);
```

Comment:
Previously the two constructors for FFmpegReader differed only in that one required an `inspect_reader` argument, which turned off the `Open(); Close();` cycle that was always performed by the one-arg version. (IOW, there was no reason to use the 2-arg version unless `inspect_reader` was false.) By making it one constructor with an optional `inspect_reader` argument that defaults to `true`, I was able to eliminate an entire constructor's worth of duplicate code.

While I was touching these constructors, I changed their string args to take a const reference, so that we can try it out as a change. Especially for arguments that are simply copied to member variables in the initializer list, there should be no down side, only a slightly more efficient method that avoids one unnecessary redundant copy of a class that's just going to be copied **again** by the initializer list. I anticipate submitting a much larger PR that updates a significant porition of our class methods in the same manner, so consider this a trial run.

Reply:
The documentation for a bunch of these constructors also kept ending on, "...throws one of the following exceptions." with no list of exceptions after it. There's nothing worse than documentation that teases you and doesn't follow through, so I changed it to not over-promise.